packages/ui/src/components/va-badge/_variables.scss

```diff
@@ -17,7 +17,7 @@
   --va-badge-text-wrapper-transition: 0.2s cubic-bezier(0.4, 0, 0.6, 1);
   --va-badge-text-wrapper-display: inline-flex;
   --va-badge-text-wrapper-border: solid 0.125rem;
-  --va-badge-text-wrapper-border-radius: 0.6rem;
+  --va-badge-text-wrapper-border-radius: 0.125rem;
```

Comment:
Isn't it a braking-change? Might be spooky to see badge suddenly changed one day. I would suggest changing it in 1.6.0 or we don't care?

Reply:
@m0ksem I think it's not that breaking change if user will mention that, he can use his own border-radius value.